bundle/schema/docs/bundle_descriptions.json

```diff
@@ -36,30 +51,45 @@
         }
       }
     },
-    "targets": {
+    "environments": {
```

Comment:
What happened here? It looks like the environments block got reinstated with empty docs.

Reply:
This is expected. `enviroments` was renamed to -> `targets` here manually https://github.com/databricks/cli/pull/670/files#diff-2b39aa846db6105db22d39143a8474f599cd77a4085521636c97f0949a82f73a

Generating the docs manually from the bundle Go struct also generated docs for for `enviroments`. However this  does not make a difference for the final generated `enviroments` schema.

Reply:
I also tested out the `deprecated` JSON schema keyword. Unfortunately the Red Hat YAML extension does not have a special UI (Like strikethough for example) for deprecated fields. 

Note, we can still introduce a `bundle: "deprecated"` tag, and fill in all the descriptions with a message like `[DEPRECATED] ...`. That'll help with migration here and any future fields we depcreate.

Reply:
Please mask it out. I'd rather have a squiggly line on `environments` than valid autocompletion.

Reply:
Done. I have marked `enviroments` as "deprecated". From this point onwards it will not generated as part of the bundle schema.